data/json/items/vehicle/frames.json

```
@@ -96,5 +96,29 @@
     "price": "10 USD",
     "price_postapoc": "50 cent",
     "copy-from": "foldframe"
+  },
+  {
+    "type": "GENERIC",
+    "id": "frame_plastic",
+    "name": { "str": "plastic frame" },
+    "description": "A large frame made of plastic.  Useful for cheap and lightweight vehicles.",
+    "weight": "2500 g",
+    "color": "brown",
+    "material": [ "plastic" ],
+    "price": "35 USD",
+    "price_postapoc": "1 USD",
+    "copy-from": "foldframe"
+  },
+  {
+    "type": "GENERIC",
+    "id": "frame_carbon",
+    "name": { "str": "carbon fiber frame" },
+    "description": "A large frame made of plastic.  Useful for lightweight yet sturdy vehicles.",
+    "weight": "2500 g",
```

Mebbe just a lil lower so it's lighter than plastic, like the difference between plastic and carbon fiber boat hulls. Went with the common 0.6x for the suggestion, contrast fiber fiber boat hulls being a third the weight of plastic ones.
```suggestion
    "weight": "1500 g",
```